hack/e2e-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Can we prefix the names of all scripts that are not intended to be used by developers but by CI only with `ci-`? So in this case that would mean move the script to `hack/ci-e2e-test.sh`